tools/sof-get-default-tplg.sh

```diff
@@ -8,5 +8,5 @@
 # sof-apl-pcm512x.tplg will be returned
 #
 
-tplg_file=$(sudo journalctl -k |grep -i topology |awk -F ':' '/tplg/ {print $NF;}'|tail -n 1)
+tplg_file=$(sudo journalctl -k -g topology |awk -F ':' '/tplg/ {print $NF;}'|tail -n 1)
```

Comment:
changes look good. 
`sudo journalctl -k -g topology` prints
```
12月 08 03:15:30 sh-glk-bob-da7219-01 kernel: sof-audio-pci-intel-apl 0000:00:0e.0: loading topology:intel/sof-tplg/sof-glk-da7219.tplg
12月 08 03:15:30 sh-glk-bob-da7219-01 kernel: sof-audio-pci-intel-apl 0000:00:0e.0: Topology: ABI 3:20:0 Kernel ABI 3:18:0
12月 08 03:15:30 sh-glk-bob-da7219-01 kernel: sof-audio-pci-intel-apl 0000:00:0e.0: warn: topology ABI is more recent than kernel
```
will it be better to change the searching string to `loading topology` or `intel/sof-tplg` or `sof-tplg`, which only matches
```
12月 08 03:15:30 sh-glk-bob-da7219-01 kernel: sof-audio-pci-intel-apl 0000:00:0e.0: loading topology:intel/sof-tplg/sof-glk-da7219.tplg
```

Reply:
> sudo journalctl -k -g topology prints... (too much stuff)

`sudo journalctl -k |grep -i topology` prints the same thing - just more slowly :-)

> will it be better to change the searching string 

Probably yes but that's a different question.